src/utils/ReactChildren.js

```diff
@@ -81,7 +81,7 @@ function mapSingleChildIntoContext(traverseContext, child, name, i) {
   var mapBookKeeping = traverseContext;
   var mapResult = mapBookKeeping.mapResult;
 
-  var keyUnique = !mapResult.hasOwnProperty(name);
+  var keyUnique = ('undefined' === typeof mapResult[name]);
```

Comment:
We use the constant (string literal) on the right hand side. Which is triggering a lint warning which is failing the unit tests.